src/table/constants.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export type ColumnId = string | symbol;
```

Comment:
Any reason this doesn't go into the interfaces file?

Reply:
No good reason - updated!